availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
I didn't say that very well. Let me try again.
print_trailer(true) or print_trailer(1)
Neither tells me anything. There is no context. All I know is, we call a function that takes one parameter that most likely tests for truthy.
I have to go look at function print_trailer($include_nav_links) to even see what the variable is named.
I would rather cut a few thousand K bytes.
Well, actually, I would rather not put them to start with. Oh, well.